bears/general/TextLintBear.py

```diff
@@ -0,0 +1,163 @@
+                        no_empty_section: bool=True,
+                        check_date_weekday_mismatch: bool=True,
+                        ginger: bool=True,
+                        max_lines_per_file: int=300,
+                        max_comma_per_sentence: int=4,
+                        check_ng_words: typed_list(str)=[],
+                        period_in_list_item: bool=True,
+                        minimum_acronym_length: int=3,
+                        maximum_acronym_length: int=5,
+                        ignore_acronyms: typed_list(str)=[],
+                        check_rousseau: bool=True,
+                        check_alex: bool=True,
+                        check_common_misspellings: bool=True,
+                        write_good: bool=True,
+                        check_invalid_link: bool=True,
+                        textlint_config: str='',
+                        ):
+        """
+        :param keyword_todo:
+            This rule checks ``- [ ]`` TODO text.
+        :param no_start_duplicated_conjunction:
+            This rule checks whether your sentence starts with a duplicated
+            conjunction.
+        :param no_empty_section:
+            This rule does not allow to create an empty section.
+            For example, there is an empty section ``# Header B`` below:
+            ```
+            # Header A
+
+            text.
+
+            # Header B
+            ```
+        :param check_date_weekday_mismatch:
+            This rule finds a mismatch between date and the corresponding
+            weekday.
+        :param ginger:
+            This rule checks your English grammar with Ginger Proofreading.
+        :param max_lines_per_file:
+            Number of lines allowed per file.
+        :param max_comma_per_sentence:
+            Number of commas allowed per sentence.
+        :param check_ng_words:
+            This rule checks NG (No Good) words. You can set these NG words
+            manually.
+        :param period_in_list_item:
+            This rule checks whether there is a period in list item.
+        :param minimum_acronym_length:
+            Minimum length for the unexpanded acronym.
+        :param maximum_acronym_length:
+            Maximum length for the unexpanded acronym.
+        :param ignore_acronyms:
+            A list that contains the acronyms to ignore.
+        :param check_rousseau:
+            This rule checks English sentence using rousseau.
+        :param check_alex:
+            This rule helps you find gender favouring, polarising, race
+            related, religion inconsiderate, or other unequal phrasing.
+        :param check_common_misspellings:
+            This rule helps to find common misspellings from Wikipedia's
+            list of common misspellings.
+        :param write_good:
+            This rule checks your English styles with
+            https://github.com/btford/write-good.
```

Comment:
I think you have to use RST syntax somehow to include links. Don't know how that works now, could you look up?